### ISSUE TITLE
anchor center is inside gt?

### ISSUE THREAD

**Issue by johnlanbor** [sfzhang15/ATSS#11]
`
                for ng in range(num_gt):
                    l = anchors_cx_per_im[candidate_idxs[:, ng]] - bboxes_per_im[ng, 0]
                    t = anchors_cy_per_im[candidate_idxs[:, ng]] - bboxes_per_im[ng, 1]
                    r = bboxes_per_im[ng, 2] - anchors_cx_per_im[candidate_idxs[:, ng]]
                    b = bboxes_per_im[ng, 3] - anchors_cy_per_im[candidate_idxs[:, ng]]
                    is_in_gts.append(torch.stack([l, t, r, b], dim=1).min(dim=1)[0] > 0.01)
`
l,t,r,b is regression target, so if we want to make sure this anchor center is in gt, we can make sure l,b is a negative number, and r,b is positive. so why this operation `torch.stack([l, t, r, b], dim=1).min(dim=1)[0] > 0.01` made min(l,t,r,b) is all positive number. That would made anchor center is in the left bottom of gt box.
Thanks a lot.

**Comment by sfzhang15** [comment]
@johnlanbor 
l, t, r, b are the distance from the anchor center to four bounds of the gt box. All of l, t, r, b are positve number means this anchor center is in gt. Please check those lines of code to understand what it means.

**Comment by johnlanbor** [comment]
Sorry, My Fault
